src/ifp/src/InitFloorplan.tcl

```diff
@@ -174,10 +174,10 @@ proc make_tracks { args } {
     set layer_name [lindex $args 0]
     set layer [$tech findLayer $layer_name]
     if { $layer == "NULL" } {
-      utl::error "IFP" 21 "layer $layer_name not found."
+      utl::error "IFP" 23 "layer $layer_name not found."
```

Comment:
for future reference there is no requirement that message IDs be sequential, just unique, so you don't need to renumber existing calls